src/jade/br/index.jade

```diff
@@ -1,140 +0,0 @@
-doctype html
```

Comment:
Shame to throw these out. I see that there's [an internationalization plugin for Astro](https://github.com/yassinedoghri/astro-i18next) which is nearly out of beta. I can very easily hook this site into our translation service and manually maintain a boatload of languages like Portuguese here. I'll create an issue to track this